gulpfile.js

```diff
@@ -223,4 +232,9 @@ gulp.task('watch', function() {
 // This task starts google app engine development server.
 // TODO(Barnabas Makonda): check if files are already generated and if so
 // do not build.
-gulp.task('start_devserver', ['build', 'gulpStartGae', 'watch']);
+var isProdMode = argv.prod_env === 'True';
+if (isProdMode) {
+  gulp.task('start_devserver', ['gulpStartGae', 'watch']);
```

Comment:
Hm, that seems a bit confusing. Is it possible to consolidate things so that all the choice of what tasks to run is here instead? Just wondering.

Also, I thought the original thing you wanted to solve was not to generate the minified files if the server is being started in dev mode? So I thought the 'build' task would be run in prod but not in dev. Let me know if I'm misunderstanding...